From 7149e5ab62e783ce7df61ea954c35f1faaf116d5 Mon Sep 17 00:00:00 2001
From: Xin Ouyang <Xin.Ouyang@windriver.com>
Date: Thu, 5 Apr 2012 13:38:57 +0800
Subject: [PATCH 2/4] refpolicy-rbac: Define default role type pairs.

The login process will read the default_context to determine the default
role/type pair to use for initial login.

And the default_type contains a list of role/type pairs that are used by
newrole

Upstream-Status: Inappropriate [no upstream]

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
---
 config/appconfig-mcs/default_contexts | 2 +-
 config/appconfig-mcs/default_type     | 4 ++++
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/config/appconfig-mcs/default_contexts b/config/appconfig-mcs/default_contexts
index 698d54c..218ec62 100644
--- a/config/appconfig-mcs/default_contexts
+++ b/config/appconfig-mcs/default_contexts
@@ -1,5 +1,5 @@
 system_r:crond_t:s0		user_r:user_t:s0 staff_r:staff_t:s0 sysadm_r:sysadm_t:s0 unconfined_r:unconfined_t:s0 user_r:cronjob_t:s0 staff_r:cronjob_t:s0 sysadm_r:cronjob_t:s0 system_r:system_cronjob_t:s0 unconfined_r:unconfined_cronjob_t:s0
-system_r:local_login_t:s0	user_r:user_t:s0 staff_r:staff_t:s0 sysadm_r:sysadm_t:s0 unconfined_r:unconfined_t:s0
+system_r:local_login_t:s0       user_r:user_t:s0 staff_r:staff_t:s0 sysadm_r:sysadm_t:s0 unconfined_r:unconfined_t:s0 technical_lead_r:technical_lead_t:s0 manager_r:manager_t:s0 senior_engineer_r:senior_engineer_t:s0 engineer_r:engineer_t:s0
 system_r:remote_login_t:s0	user_r:user_t:s0 staff_r:staff_t:s0 unconfined_r:unconfined_t:s0
 system_r:sshd_t:s0		user_r:user_t:s0 staff_r:staff_t:s0 sysadm_r:sysadm_t:s0 unconfined_r:unconfined_t:s0
 system_r:sulogin_t:s0		sysadm_r:sysadm_t:s0
diff --git a/config/appconfig-mcs/default_type b/config/appconfig-mcs/default_type
index 33528d6..a47aa43 100644
--- a/config/appconfig-mcs/default_type
+++ b/config/appconfig-mcs/default_type
@@ -4,3 +4,7 @@ sysadm_r:sysadm_t
 staff_r:staff_t
 unconfined_r:unconfined_t
 user_r:user_t
+engineer_r:engineer_t
+senior_engineer_r:senior_engineer_t
+manager_r:manager_t
+technical_lead_r:technical_lead_t
-- 
1.9.1

